### ISSUE TITLE
Auto build for future updates?

### ISSUE THREAD

**Issue by zap-py** [hippuli/Bagnon#1]
Hey, thanks for putting up the repo. Will this be updated and packaged automatically when the source repo publishes changes? I'd love to add your repo as a GitHub source to my WowUp installation.

**Comment by hippuli** [comment]
Hey, these wow addon repos are 1:1 copies from the release files on curse and I will be updating them when there are new releases. It is a semi-automated process, so it won't be instant (I get notifications on releases, but the actual re-packaging needs couple of manual steps)
For the bagnon itself, there was an update 5 days ago in curse but I decided to skip it because of multiple problems reported on curse comments (guildbank shift clicking disabled and version number went from 9.2.0 to 9.1.5 which is very strange)

tldr: yes these will be updated semi-automatically based on curse releases

**Comment by zapp2me** [comment]
Awesome, thanks!
